app/views/staff/shared/xml/_activity.xml.haml

```diff
@@ -16,16 +16,16 @@
     %participating-org{"ref" => activity.extending_organisation.iati_reference, "type" => activity.extending_organisation.organisation_type, "role" => 3}
       %narrative= activity.extending_organisation.name
   %activity-status{code: "#{activity.status}"}/
-  %activity-date{"iso-date" => "#{activity.planned_start_date}", type: "1"}/
-  %activity-date{"iso-date" => "#{activity.planned_end_date}", type: "2"}/
-  %activity-date{"iso-date" => "#{activity.actual_start_date}", type: "3"}/
-  %activity-date{"iso-date" => "#{activity.actual_end_date}", type: "4"}/
-  %recipient-region{"code" => "#{activity.recipient_region}"}/
-  %sector{"vocabulary" => "1", "code" => "#{activity.sector}"}/
-  %default-flow-type{"code" => "#{activity.flow}"}/
-  %default-finance-type{"code" => "#{activity.finance}"}/
-  %default-aid-type{"code" => "#{activity.aid_type}"}/
-  %default-tied-status{"code" => "#{activity.tied_status}"}/
+  %activity-date{"iso-date" => activity.planned_start_date, type: "1"}/
```

Comment:
Is the xml vaild with `"iso-date => ,`?

Reply:
If the value is blank then the attribute name won't show up. Before it was adding `iso-date=""` to the XML which was being parsed by the validator as an invalid date.